src/openms/include/OpenMS/FILTERING/DATAREDUCTION/MassTraceDetection.h

```diff
@@ -105,7 +105,7 @@ namespace OpenMS
         typedef std::multimap<double, std::pair<Size, Size> > MapIdxSortedByInt;
```

Comment:
Is the typedef still needed?

Reply:
no thanks, I am thinking about adding a new one